p2p/message.go

```diff
@@ -124,58 +113,63 @@ var dataExcludedMsgs = map[uint64]struct{}{
 	0x10: empty, //Receipts
 }
 
-func SendEthSubproto(w MsgWriter, msgcode uint64, data interface{}, peers ...discover.NodeID) error {
+func SendEthSubproto(w MsgWriter, msgcode uint64, data interface{}, connInfoCtx ...interface{}) error {
```

Comment:
When sending our status, the data is logged as `struct {}{}`. The size (always 76) is logged correctly, so I think the data passed in is fine. Not a big concern for now as we don't really care about our status that's sent.

Reply:
Is this only a logging issue? It seems like it only affects the logging of the data, and not any logging metadata.

Reply:
Yes. It only affects content of the data.

Reply:
I think this shouldn't be a problem for us, since our focus is not on the content of the messages, more the messages themselves.

Reply:
Resolved. It had nothing to do with before/after sending; just a stupid mistake. I'm leaving it as logging after sending, but `time.Now()` will be moved to before.